explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -74,198 +74,191 @@ object SiderealTargetEditor {
 
   implicit val propsReuse: Reusability[Props] = Reusability.derive
 
-  val AladinRef = AladinCell.component
-
-  class Backend() {
-    def render(props: Props) =
-      AppCtx.using { implicit appCtx =>
-        val undoCtx     = UndoContext(props.undoStacks, props.target)
-        val target      = props.target.get
-        val undoViewSet = UndoView(props.id, undoCtx)
-
-        val allView = undoViewSet(
-          Iso.id.asLens,
-          { t: SiderealTarget =>
-            EditSiderealInput.name.replace(t.name.assign) >>>
-              TargetQueries.UpdateSiderealTracking(t.tracking) >>>
-              TargetQueries.replaceMagnitudes(t.magnitudes)
-          }
-        )
+  val component =
+    ScalaFnComponent
+      .withHooks[Props]
+      .renderWithReuse { props =>
```

Comment:
You should be able to do`ScalaFnComponent.withReuse[Props]` if there are no hooks.

Reply:
Oh didn't know that